doc/parameters.md

```diff
@@ -408,7 +408,7 @@ Path to a file with the X.509 certificate secret key in the PEM format.
 **Values:** positive integers
 
 Defines the maximum length of the client certificate chain.
-If this parameter has 1 as its value, it implies that this length might include one additional certificate (eg. intermediate CA).
+If this parameter has 1 as its value, it implies that this length might include one additional certificate (eg. root CA).
```

Comment:
```suggestion
If this parameter has `1` as its value, it implies that this parameter includes an additional certificate. For example,  root CA.
```

Reply:
Thanks, I changed it based on your suggestion.